SAMtags.tex

```diff
@@ -75,32 +76,36 @@ \section{Standard tags}
   {\tt H1} & i & Number of 1-difference hits (see also {\tt NM}) \\
   {\tt H2} & i & Number of 2-difference hits \\
   {\tt HI} & i & Query hit index \\
-  {\tt IH} & i & Number of stored alignments in SAM that contains the query in the current record\\
+  {\tt IH} & i & Number of stored alignments in SAM that contains the query in the current record \\
   {\tt LB} & Z & Library \\
-  {\tt MC} & Z & CIGAR string for mate/next segment\\
+  {\tt MC} & Z & CIGAR string for mate/next segment \\
   {\tt MD} & Z & String for mismatching positions \\
   {\tt MF} & ? & Reserved for backwards compatibility reasons \\
+  {\tt MI} & Z & Molecular-barcode sequence or identifier \\
```

Comment:
Can I suggest MI be defined with: "Molecular identifier; a string which uniquely identifies the molecule within the library".  The way we're using this tag, the value is file-wide or at least library-wide unique, which UMI sequenced tend not to be, as you often combine them with other info (read start/stop, proximity to other reads with the same UMI) to infer molecular identity.